site/src/_pages/docs/4_faq.mdx

```diff
@@ -191,7 +191,7 @@ We don’t prescribe canonical types for any _thing_, but do recommend that user
 
 </FAQ>
 
-<FAQ question={<div>How does the Block Protocol relate to <i>schema.org</i>?</div>}>
+<FAQ question={<span>How does the Block Protocol relate to <i>schema.org</i>?</span>}>
```

Comment:
Prevents `<div>`s inside `<p>`s on `/docs/faq`. Fixes a [hydration error on Sentry](https://sentry.io/organizations/hashintel/issues/3476303370/events/8c51d2bce8a94763a8f85529f272084c/?project=6603021) that still remains after #497